src/lib/components/WebvizPluginPlaceholder/WebvizPluginPlaceholder.tsx

```diff
@@ -75,7 +80,26 @@ const WebvizPluginPlaceholder = (
         else {
             didMountRef.current = true;
         }
-    });
+        showDeprecationWarnings();
+    }, []);
+
+    const showDeprecationWarnings = () => {
+        for (const warning of deprecation_warnings) {
+            enqueueSnackbar(
+                warning.message,
+                {
+                    variant: "warning",
+                    action: () => (
+                        <Button onClick={() => window.open(warning.url, '_blank')}>More info</Button>
```

Comment:
```suggestion
                        <a href={warning.url} target={"_blank"}>More info</a>
```
Would this shorter variant work, or do we need button?